.github/workflows/ci__minimal.yaml

```diff
@@ -9,6 +9,7 @@ on:
     workflow_dispatch: ~
     push:
         branches-ignore:
+            - 'dependabot/**'
```

Comment:
I know it could be pushed in a separate PR, but this one is so small, that I'll be waiting longer for CI than it's worth.